indico/modules/rb/client/js/modules/calendar/actions.js

```diff
@@ -109,7 +113,7 @@ export function fetchCalendar(fetchRooms = true) {
   };
 }
 
-export function fetchActiveBookings(limit, fetchRooms = true) {
+export function fetchActiveBookings(limit, fetchRooms = true, range = null) {
```

Comment:
You have access to `state` in this function, so you do not need to pass this from the outside